client/src/sections/Login/LoginModal.tsx

```diff
@@ -0,0 +1,40 @@
+import {Modal, TextField} from '@shopify/polaris';
+import React from 'react';
+
+export default function LoginModal(props: any) {
+
+  return (
+    <div className='loginModal'>
+      <Modal
+        open={props.active}
+        onClose={props.handleChange}
+        title="Login"
+        primaryAction={{
+          content: 'Login',
+          onAction: props.handleLogin,
```

Comment:
I'm looking at all these handlers for the fields, and I think these states can simply be moved into this modal component itself (as opposed to being passed down from the parent component).